crates/floresta-chain/src/pruned_utreexo/consensus.rs

```diff
@@ -238,13 +254,64 @@ impl Consensus {
             ),
         }
     }
-    #[allow(unused)]
+    pub fn validate_block_time(
+        block_timestamp: u32,
+        mtp: u32,
+        time: impl NodeTime,
+    ) -> Result<(), BlockValidationErrors> {
+        let its_too_old = mtp > block_timestamp;
+        let its_too_new = block_timestamp > (time.get_time().sub(2 * HOUR));
```

Comment:
Shouldn't this be plus two hours? A block can't be more than two hours into the future.